test/hotspot/jtreg/testlibrary/ctw/Makefile

```diff
@@ -47,7 +47,7 @@ LIB_FILES = $(shell find $(TESTLIBRARY_DIR)/jdk/test/lib/ \
     $(TESTLIBRARY_DIR)/jdk/test/lib/util \
     $(TESTLIBRARY_DIR)/jtreg \
     -maxdepth 1 -name '*.java')
-WB_SRC_FILES = $(shell find $(TESTLIBRARY_DIR)/sun/hotspot $(TESTLIBRARY_DIR)/jdk/test/whitebox -name '*.java')
+WB_SRC_FILES = $(shell find $(TESTLIBRARY_DIR)/jdk/test/lib/compiler $(TESTLIBRARY_DIR)/jdk/test/whitebox -name '*.java')
```

Comment:
I think `$(TESTLIBRARY_DIR)/sun/hotspot` should be removed.
[JDK-8067223](https://bugs.openjdk.org/browse/JDK-8067223) left copy of WB there because some tests were still using it. But recent @coleenp changes [JDK-8271707](https://bugs.openjdk.org/browse/JDK-8271707) and followed [JDK-8275662](https://bugs.openjdk.org/browse/JDK-8275662) removed duplicated WB in `sun/hotspot`.

Reply:
NM that. I looked on description in JBS and saw that you need CompilerUtils.java which is in `jdk/test/lib/compiler`.
I will run our internal testing.

Reply:
I looked and `ClassTransformer.java` (which needs `CompilerUtils.java`) is recent addition: [JDK-8240908](https://bugs.openjdk.org/browse/JDK-8240908)
That is why CTW did not need `CompilerUtils.java` before.